paper.md

```diff
@@ -0,0 +1,84 @@
+---
+
+# Summary
+
+Qiskit Dynamics is an open source Python library for numerically simulating the time dynamics of finite-dimensional quantum systems. The goal of the package is to provide flexible configuration of the numerical methods used for simulation: general tools for transforming models of quantum systems for more efficient simulation (rotating frames and the rotating wave approximation), choice of array representations (dense v.s. sparse, and different array libraries), and access to different types of underlying solvers (standard ODE v.s. geometric solvers). The package also contains advanced functionality for computing time-dependent perturbation theory expressions used in robust quantum control optimization `[@perturb1; @perturb2]`. 
+
+As part of the Qiskit Ecosystem (https://qiskit.org/ecosystem), the package interfaces with other parts of Qiskit `[@Qiskit]`. Most notably, Qiskit Dynamics provides tools for simulating control sequences specified by Qiskit Pulse `[@alexander_qiskit_2020]`, which is used to specify hardware-level control of quantum computers. Higher level interfaces allow users to build and interact with simulation-based objects that target the same constraints (coupling map, timing, etc.) as a specified IBM Quantum computer.
+
+Lastly, to facilitate high-perfomance applications, Qiskit Dynamics is compatible with the JAX array library `[@jax2018github]`. As such, all core computations are just-in-time compilable, automatically differentiable, and executable on GPU.
+
+# Statement of need
+
+Numerical simulation of time-dependent quantum systems is a useful tool in quantum device characterization and design, as well as control optimization. As these applications often involve the expensive process of repeatedly simulating a system across different parameters (e.g. in exploratory parameter scans, or in optimizations), it is important for users to be able to easily select the numerical methods that are most performant for their specific problem. The ability to automatically differentiate and compile simulations is also critical for flexible control optimization research. 
+
+Furthermore, having a simulation-based drop-in replacement for real quantum computing systems is useful for developers building software tools for low-level control of experiments, such as Qiskit Pulse `[@alexander_qiskit_2020]` and Qiskit Experiments `[@kanazawa_qiskit_2023]`.
+
+# Related open source packages
+
+Due to its importance, many open source packages contain time-dependent quantum system simulation tools. In Python, these include QuTiP `[@qutip]`, TorchQuantum `[@torchquantum]`, and C3 `[@C3]`. C++ packages (also with Python interfaces) include lindbladmpo `[@lindbladmpo]` and Quandary `[@quandary]`. Packages also exist in other languages, such as the Hamiltonian open quantum system toolkit (HOQST) `[@hoqst]` and a Framework for Quantum Optimal Control `[@julia_qc]` in Julia, and Spinach `[@spinach]` in MATLAB.
```

Comment:
A few words on how we differentiate?

Reply:
I assume here you mean "how do we compare"? My response here is roughly the same as in the previous point: is there any need to make comparisons?